configuration/api.go

```diff
@@ -114,6 +114,75 @@ func (a *API) GetBot(id string) (*BotAgentDetails, error) {
 	return resp.BotAgent, err
 }
 
+// CreateAgent creates a new Agent with specified parameters within a license
+func (a *API) CreateAgent(agent *Agent) (string, error) {
+	var resp createAgentResponse
+	err := a.Call("create_agent", agent, &resp)
+
+	return resp.ID, err
+}
+
+// GetAgent returns the info about an Agent specified by id
+func (a *API) GetAgent(id string, fields []string) (*Agent, error) {
+	var resp getAgentResponse
+	err := a.Call("get_agent", &getAgentRequest{
+		ID:     id,
+		Fields: fields,
+	}, &resp)
+
+	return resp, err
+}
+
+// ListAgents returns all Agents within a license.
```

Comment:
please unify dot at end of the comments across whole document (dot should be used everywhere)